doc/developers/run_app.rst

```diff
@@ -2,7 +2,73 @@
 Running CCF Applications
 ========================
```

Comment:
That should be renamed to Sandbox in my opinion, and expanded to cover the various usage scenarios it addresses.